_Week5/group4/demonstration.do

```diff
@@ -0,0 +1,12 @@
+global username "/Users/geenapanzitta/Library/CloudStorage/Box-Box/"
+
+cd "${username}ECON490/Week5/"
+
+clear all 
+run program.do
+
+graphsum auto price mpg
+
+graphsum census death divorce
+
+graphsum surface latitude temperature

```

Comment:
Nice examples! Would be great if you added a list of the system data sets the program can operate on.

Reply:
That's more related to the sysuse command, but I added a small comment on the readme that explains how to find the available datasets.